1_fetch/src/01_get_data.R

```diff
@@ -1,7 +1,7 @@
 # this function takes an output directory, and downloads the mendota csv file there.
 
-get_data = function(project_output_dir) {
-  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+get_data = function(out_dir) {
```

Comment:
I think instead of `out_dir`, `out_file` might be a better argument name since it is the complete filepath, not just the directory.